setup.py

```diff
@@ -1,41 +1,46 @@
-#!/usr/bin/env python
 from setuptools import setup, find_packages
 import opmd_viewer  # In order to extract the version number
 
+# Get the long description
+# If possible, use pypandoc to convert the README from Markdown
+# to reStructuredText, as this is the only supported format on PyPI
+try:
+    import pypandoc
+    long_description = pypandoc.convert( './README.md', 'rst')
+except (ImportError, RuntimeError):
+    long_description = open('./README.md').read()
 # Get the package requirements from the requirements.txt file
-with open('requirements.txt') as f:
+with open('./requirements.txt') as f:
     install_requires = [line.strip('\n') for line in f.readlines()]
 # Since wget cannot be installed with conda, it is added separately here
 install_requires.append('wget')
 
 # Main setup command
 setup(name='openPMD-viewer',
       version=opmd_viewer.__version__,
+      description='Visualization tools for openPMD files',
+      long_description=long_description,
       maintainer='Remi Lehe',
       maintainer_email='remi.lehe@lbl.gov',
-      description='Visualization tools for openPMD files',
-      url='git@bitbucket.org:berkeleylab/opmd_viewer.git',
+      license='BSD 3-Clause License',
+      url='https://github.com/openPMD/openPMD-viewer.git',
       packages=find_packages('./'),
       package_data={'opmd_viewer': ['notebook_starter/*.ipynb']},
       scripts=['opmd_viewer/notebook_starter/openPMD_notebook'],
       install_requires=install_requires,
       tests_require=['pytest', 'jupyter'],
       setup_requires=['pytest-runner'],
-      platforms='any',
       classifiers=[
           'Programming Language :: Python',
           'Development Status :: 4 - Beta',
           'Natural Language :: English',
           'Environment :: Console',
-          'Environment :: Jupyter',
           'Intended Audience :: Science/Research',
-          'License :: BSD-3-Clause-LBNL',
+          'License :: OSI Approved :: BSD License',
```

Comment:
yes I know you like to change this line (and `license` above) to a pre-defined identifier but it's actually not 100% correct.
the LBNL (and LLNL) BSD-style licenses are a 3-Clause BSD license *but* with an extensions reserving them to use improvements for a certain amount of time. I think we should better be transparent about this before someone complains :)

Also, it will still work ok enough. See Forthon:
- [pypi](https://pypi.python.org/pypi/Forthon)
- [badge](https://github.com/dpgrote/Forthon/blob/master/README.md)